impl/client/src/main/javascript/web/pentaho/type/list.js

```diff
@@ -208,6 +208,53 @@ define([
         return clone;
       },
 
+      // region configuration
+
+      /**
+       * Configures a list instance with a given configuration.
+       *
+       * When `config` is another list, behaviour is currently undefined.
```

Comment:
I would say that it is not implemented and that it throws an error. It is undefined in our heads (thus not implemented) but not in the code. This can make someone to think it is "ok" to pass a list, you just don't know what will happen.